src/generators/default_map_generator_job.cpp

```diff
@@ -814,22 +808,21 @@ std::string default_map_generator_job::default_generate_map(size_t width, size_t
 
 	t_translation::tstarting_positions starting_positions;
 	LOG_NG << output_map(terrain, starting_positions);
-	LOG_NG << "placed land forms\n";
-	LOG_NG << (SDL_GetTicks() - ticks) << "\n"; ticks = SDL_GetTicks();
-
-	// Now that we have our basic set of flatland/hills/mountains/water,
-	// we can place lakes and rivers on the map.
-	// All rivers are sourced at a lake.
-	// Lakes must be in high land - at least 'min_lake_height'.
-	// (Note that terrain below a certain altitude may be made
-	// into bodies of water in the code above - i.e. 'sea',
-	// but these are not considered 'lakes', because
-	// they are not sources of rivers).
-	//
-	// We attempt to place 'max_lakes' lakes.
-	// Each lake will be placed at a random location,
-	// if that random location meets the minimum terrain requirements for a lake.
-	// We will also attempt to source a river from each lake.
+	LOG_NG << "Placed landforms. " << (SDL_GetTicks() - ticks) << " ticks elspased" << "\n";
```

Comment:
I personally think these two output statements were better as separate lines. Also, you misspelled "elapsed".